docker-compose.yml

```diff
@@ -1,11 +1,20 @@
-version: '3.8'
+version: "3.8"
 
 services:
   postgres:
     image: library/postgres:12
     ports:
-    - 5432:5432
+      - 5432:5432
     environment:
       POSTGRES_PASSWORD: password
       POSTGRES_USER: beacons-service
-      POSTGRES_DB: beacons
\ No newline at end of file
+      POSTGRES_DB: beacons
+
+  beacons-service:
+    build:
+      context: ./
+    container_name: app
+    ports:
```

Comment:
You will also need to add some environment variables here, otherwise the beacons service will not be able to talk to PostgreSQL if they are both running as containers in the same Docker network.

The environment variable required is: `SPRING_DATASOURCE_URL: jdbc:postgresql://postgres:5432/beacons`